fitting/dijetfit.py

```diff
@@ -541,6 +555,10 @@ def fitting_options():
                       help="Where to put the plots")
     parser.add_option("-l", "--label", dest="label", default='test',
                       help="Label for file names")
+    parser.add_option("--draw_sig", dest="draw_sig", default=False,
+                      action = 'store_true', help="Draw separate signal contribution on S+B fit plots")
+    parser.add_option("--no_draw_sig", dest="draw_sig", action = 'store_false', help="Don't draw separate signal and bkg contribution on S+B fit plots")
+    parser.set_defaults(draw_sig = True)
```

Comment:
isn't the `--draw_sig` redundant here?

Reply:
Ah yeah, I changed the default option so we don't need it anymore. I'll remove it